backend/src/v5/utils/quota.js

```diff
@@ -35,7 +35,8 @@ Quota.getQuotaInfo = async (teamspace) => {
 		// paypal subs have a different schema - and no oen should have an active paypal sub. Skip.
 		if (key !== 'paypal') {
 			userHasHadPaidPlan = true;
-			const { expiryDate, data, collaborators: subCollaborators } = subs[key];
+			const { expiryDate, data } = subs[key];
```

Comment:
You should be able to do this:

```suggestion
			const { expiryDate, data, collaborators: subCollaborators = 0 } = subs[key];
```